packages/http-logger/types/index.d.ts

```diff
@@ -0,0 +1,18 @@
+import { RequestHandler, Request } from "express";
+
+/**
+ * @description Options for logging middleware function
+ */
+type HttpLoggerOptions = {
+  whitelists?: string[];
+  loggable?: string[] | ((req: Request) => Record<string, any>[]);
```

Comment:
Since you've mentioned the entire express `Request` type here, shall we update the actual code to pass in the entire req as well, right now it's just passing the headers and body. Passing just the 2 of them seems pointless once i think of it again now. Also shall we add express to peer deps of this package.json file?

Reply:
Okay will do.

Reply:
Done

Reply:
@Akalanka47000 can you merge this.